configure.py

```diff
@@ -1785,14 +1779,28 @@ def icu_download(path):
       else:
         print(f'Re-using existing {targetfile}')
       if targetfile.is_file():
-        print(f'Checking file integrity with {hashAlgo}:\r')
-        gotHash = nodedownload.checkHash(targetfile, hashAlgo)
-        print(f'{hashAlgo}:      {gotHash}  {targetfile}')
-        if expectHash == gotHash:
-          return targetfile
-
-        warn(f'Expected: {expectHash}      *MISMATCH*')
-        warn(f'\n ** Corrupted ZIP? Delete {targetfile} to retry download.\n')
+        if "gpg" in icu:
```

Comment:
Should this also verify that gpg is present and installed?

Reply:
@srl295 

Changed to check both gpg and md5.
- Only warn when md5 cannot be obtained.
- Only warn if gpg signature cannot be obtained or gpg is not installed
- If neither is available, warn and no update.
- If either fails verification, warn out and no update